src/PowerShellEditorServices/Services/Symbols/PesterDocumentSymbolProvider.cs

```diff
@@ -59,8 +59,10 @@ private static bool IsPesterCommand(CommandAst commandAst)
                 return false;
             }
 
-            // Ensure the first word is a Pester keyword
-            if (!PesterSymbolReference.PesterKeywords.ContainsKey(commandAst.GetCommandName()))
+            // Ensure the first word is a Pester keyword and in Pester-module if using module-qualified call
+            string commandName = CommandHelpers.StripModuleQualification(commandAst.GetCommandName(), out ReadOnlyMemory<char> module);
+            if (!PesterSymbolReference.PesterKeywords.ContainsKey(commandName) ||
+                !module.Span.Equals("pester".AsSpan(), StringComparison.OrdinalIgnoreCase))
```

Comment:
```suggestion
                (!module.IsEmpty && !module.Span.Equals("pester".AsSpan(), StringComparison.OrdinalIgnoreCase)))
```

We are SO BAD @SeeminglyScience. The `IsEmpty` is because it might _not_ be module qualified, it wasn't a guard!

Reply:
I guess we all learned something today 👏🥳